tests/fv/test_diags.py

```diff
@@ -0,0 +1,7 @@
+import sh
+
+
+def test_diags():
```

Comment:
As below, does this actually get run?  Shouldn't it be part of a unittest class?